src/miner_test.erl

```diff
@@ -358,14 +358,12 @@ pmap(F, L, Timeout) ->
                          end),
               N+1
       end, 0, L),
-    Ref = erlang:send_after(Timeout, self(), boom),
     L2 = [receive
               {pmap, N, R} ->
-                  {N,R};
-              boom ->
+                  {N,R}
+          after Timeout ->
```

Comment:
this seems weird?  just make the timeout longer?  That's all you're really doing, just in an uncontrolled way.

Reply:
I actually think it's a race with the timeout not being flushed, I guess we can try cancel_timer with the flush flag